### PR TITLE
Update runner Docker instructions

### PR DESCRIPTION
**Pull request by glenjamin**
# Description
To sync with https://github.com/CircleCI-Public/runner-preview-docs/blob/a440ad32b02a617bc2e8a43bb2cf184f911b2770/INSTALL-docker.md

### CODE REVIEW

File: jekyll/_cci2/runner-installation.adoc [hunk 1]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
 
-```sh
-CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_
+```bash
+docker build --file ./Dockerfile.runner.extended .
 ```
+
+=== Start the Docker container
+
+Note that we don't pass the environment variable values to the `docker` command, so they're not visible in `ps` output.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-installation.adoc:478]:
```suggestion
NOTE: CircleCI does not pass the environment variable values to the `docker` command, so these environment variables are not visible in `ps` output.
```

File: jekyll/_cci2/runner-installation.adoc [hunk 2]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
 
 NOTE: The default value is 5 hours.
 
-=== Docker Installation
+== Docker Installation
 
-The `circleci/runner` image has been created by CircleCI to work with the CircleCI runner feature. Currently the `launch-agent` tag can be used to run the launch agent in a container co-located with the CircleCI job (for example, the job will run inside the same container). In the future, a variant will be added that runs the launch agent on the host and will run each job inside its own container, via the task agent.
+The host needs to have Docker installed. Once the `runner` container is started, it will immediately attempt to start running jobs.  The container will be reused to run more jobs indefinitely until it's stopped.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-installation.adoc:453]:
```suggestion
The host needs to have Docker installed. Once the `runner` container is started, the container will immediately attempt to start running jobs. The container will be reused to run more jobs indefinitely until it is stopped.
```

File: jekyll/_cci2/runner-installation.adoc [hunk 3]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
 
-==== Getting Started
+The number of containers running in parallel on the host is constrained by the host's available resources and your jobs' performance requirements.
 
-To perform a docker pull of the CircleCI runner, run the following command:
+=== Create a Dockerfile that extends the CircleCI Runner image
+
+Here we install python3 on top of the base image
```
Comment (thomasr8) [inline jekyll/_cci2/runner-installation.adoc:459]:
```suggestion
In this example, python3 is installed on top of the base image.
```

**Review by thomasr8 — COMMENTED** [review]
Just a few minor comments you may wish to incorporate.

File: jekyll/_cci2/runner-installation.adoc [hunk 4]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
+
+=== Stopping the Docker container
+
+``` bash
+docker stop <container-name>
+```
+
+=== Additional Resources
+
+- [CircleCI Runner Image on Docker Hub](https://github.com/CircleCI-Public/runner-preview-docs/)
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc:494]:
```suggestion
- https://github.com/CircleCI-Public/runner-preview-docs/[CircleCI Runner Image on Docker Hub]
```

File: jekyll/_cci2/runner-installation.adoc [hunk 5]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
+- [CircleCI Runner Image on Github](https://github.com/CircleCI-Public/circleci-runner-docker)
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc:495]:
```suggestion
- https://github.com/CircleCI-Public/circleci-runner-docker[CircleCI Runner Image on Github]
```

File: jekyll/_cci2/runner-installation.adoc [hunk 6]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
+- [CircleCI Docs](https://circleci.com/docs/) - The official CircleCI Documentation website.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc:496]:
```suggestion
- https://circleci.com/docs/[CircleCI Docs - The official CircleCI Documentation website]
```

File: jekyll/_cci2/runner-installation.adoc [hunk 7]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
+- [Docker Docs](https://docs.docker.com/)
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc:497]:
```suggestion
- https://docs.docker.com/[Docker Docs]
```

File: jekyll/_cci2/runner-installation.adoc [hunk 8]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
+
+``` bash
+docker stop <container-name>
+```
+
+=== Additional Resources
+
+- https://github.com/CircleCI-Public/runner-preview-docs/[CircleCI Runner Image on Docker Hub]
+- https://github.com/CircleCI-Public/circleci-runner-docker[CircleCI Runner Image on Github]
+- [CircleCI Docs](https://circleci.com/docs/) - The official CircleCI Documentation website.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation.adoc]:
```suggestion
- https://circleci.com/docs/[CircleCI Docs] - The official CircleCI Documentation website.
```

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/runner-installation.adoc [hunk 9]
```diff
@@ -448,20 +448,50 @@ Here are few valid examples:
...
 
-```sh
-CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_
+```bash
+docker build --file ./Dockerfile.runner.extended .
 ```
+
+=== Start the Docker container
+
+NOTE: The environment variable values are not to the `docker` command, so these environment variables are not visible in `ps` output.
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:478]: missing word:
> NOTE: The environment variable values are not **passed** to the `docker` command, so these environment variables are not visible in `ps` output.

**Pull request merged**